components/product-details/index.js

```diff
@@ -3,51 +3,55 @@ import personData from '../../mock/person.json';
 import PropTypes from 'prop-types';
 import path from 'path';
 import Image from 'next/image';
+import Link from 'next/link';
+import { Footer } from '../../components/footer';
+import styles from './product-details.module.css';
+
+const renderUses = (productJSON) =>
+  productJSON.usage.map((use, id) => <p key={id}> {use} </p>);
+
+const renderThumbImage = (productJSON) => (
+  <img src={path.join('/assets', productJSON.image)} alt={productJSON.id} />
+);
 
 export const ProductDetails = ({ productJSON }) => {
   return (
     <>
       <NavBar personData={personData} />
-      <div className="product-container">
-        <div className="product-image">
+      <div className={styles.details}>
+        <div className={styles.bigImg}>
           <Image
+            width={600}
+            height={319}
             src={path.join('/assets', productJSON.image)}
             alt={productJSON.id}
-            width={200}
-            height={200}
+            layout="responsive"
           />
         </div>
 
-        <div className="product-details">
-          <h2> {productJSON.name}</h2>
-          <h3> Where to use</h3>
-          <ul>
-            {productJSON.usage.map((use, index) => {
-              return <li key={index}>{use}</li>;
-            })}
-          </ul>
+        <div className={styles.box}>
+          <div className={styles.row}>
+            <h2>{productJSON.name}</h2>
+            <span>{productJSON.price} Gold Coins</span>
+
+            <h3 className={styles.description}>Description</h3>
+            <p>{renderUses(productJSON)}</p>
+
+            <div className={styles.thumb}>
+              {renderThumbImage(productJSON)}
+              {renderThumbImage(productJSON)}
+              {renderThumbImage(productJSON)}
```

Comment:
Why is this repeated?

Reply:
As per my design i am showing three views/ images of the product.

Reply:
If you are repeating code, please use an array :)

Reply:
Did the changes